test/suites/storage/pvc-resize.robot

```diff
@@ -28,9 +28,12 @@ Increase Running Pod PV Size
 *** Keywords ***
 Test Case Setup
     [Documentation]    Prepare the cluster env and test pod workload.
+    ${ns}=    Create Unique Namespace
+    Set Test Variable    \${NAMESPACE}    ${ns}
     Oc Create    -f ${SOURCE_POD} -n ${NAMESPACE}
     Named Pod Should Be Ready    ${POD_NAME_STATIC}
 
 Test Case Teardown
     [Documentation]    Clean up test suite resources
     Oc Delete    -f ${SOURCE_POD} -n ${NAMESPACE}
+    Oc Delete    ns ${NAMESPACE}
```

Comment:
Maybe use this one? https://github.com/openshift/microshift/blob/main/test/resources/kubeconfig.resource#L82

Reply:
Oh thanks! I didn't see that